### ISSUE TITLE
Differences to react-geosuggest

### ISSUE THREAD

**Issue by ro-ka** [bh5-js/gmaps-places-autocomplete#1]
Hey, I’m the author of [react-geosuggest](https://github.com/ubilabs/react-geosuggest).

Just saw this lib and was wondering what the differences are?

**Comment by euforic** [comment]
@ro-ka 
- updated to use more es6 and a few other minor changes.
- Also was using in a client project so had a few css issues that had to be fixed.

 If you are ok with changes I would be happy to put in a PR to your repo.

**Comment by ro-ka** [comment]
Would be great to have a PR! No need for two almost identical plugins, I think.

**Comment by ericclemmons** [comment]
I stopped over here to ask something similar, mainly since there are no docs or examples for this project.

**Comment by ro-ka** [comment]
Yes. The CSS issues can be resolved easily when @euforic tells us what these were.

And we are improving the component all the time. Switched the task runner to npm some time ago, so using ES6 via Babel is no big issue anymore.
